plugins/woocommerce-admin/client/analytics/report/taxes/table.js

```diff
@@ -143,7 +143,16 @@ class TaxesReportTable extends Component {
 		const currency = getCurrencyConfig();
 		return [
 			{
-				label: _n( 'tax code', 'tax codes', taxesCodes, 'woocommerce' ),
+				label: _n( 'tax', 'taxes', taxesCodes, 'woocommerce' ),
```

Comment:
Should it use `totalResults` instead of `taxesCodes`?

Placing two orders with the same tax name but different rates can get this result:

![image](https://github.com/user-attachments/assets/c7c025e4-f937-4769-800b-aca17c524eaa)